server/src/main/java/org/opensearch/action/admin/indices/stats/CommonStatsFlags.java

```diff
@@ -67,6 +67,7 @@ public class CommonStatsFlags implements Writeable, Cloneable {
     // Used for metric CACHE_STATS, to determine which caches to report stats for
     private EnumSet<CacheType> includeCaches = EnumSet.noneOf(CacheType.class);
     private String[] levels = new String[0];
+    private boolean aggregateNodeIndicesStatsResponsesOnLevel = false;
```

Comment:
I think `aggregateNodeIndicesStatsResponsesOnLevel` is a better representation on the variable name. This is because aggregations on Tiered caching is also based on level and this might become misleading if we generalise it.